compiler/rustc_incremental/src/persist/fs.rs

```diff
@@ -133,24 +133,28 @@ const QUERY_CACHE_FILENAME: &str = "query-cache.bin";
 // case-sensitive (as opposed to base64, for example).
 const INT_ENCODE_BASE: usize = base_n::CASE_INSENSITIVE;
 
+/// Returns the path to the current session's dependency graph.
 pub fn dep_graph_path(sess: &Session) -> PathBuf {
     in_incr_comp_dir_sess(sess, DEP_GRAPH_FILENAME)
 }
+/// Returns the path to the current session's staging dependency graph.
 pub fn staging_dep_graph_path(sess: &Session) -> PathBuf {
     in_incr_comp_dir_sess(sess, STAGING_DEP_GRAPH_FILENAME)
 }
+/// Returns the path to the dependency graph directory.
 pub fn dep_graph_path_from(incr_comp_session_dir: &Path) -> PathBuf {
     in_incr_comp_dir(incr_comp_session_dir, DEP_GRAPH_FILENAME)
 }
```

Comment:
What is the difference between the return values of these functions? Can you expand on this in the doc comments?

Reply:
I removed references to "the current session". The signature tells the story, here, it seems.

How do these look to you, @cjgillot?

https://github.com/rust-lang/rust/blob/cac82a28bd7a780b86a581648685f439a76153ee/compiler/rustc_incremental/src/persist/fs.rs#L136-L155

Reply:
I'm not sure how to describe the query cache here.